priv/static/swagger/swagger_v2.yaml

```diff
@@ -26,6 +26,51 @@ components:
         pattern: (txi|gen):\d+(-\d+)?
         type: string
   schemas:
+    Activity:
+      description: Name Transfer
```

Comment:
same

Reply:
Just as a reminder, this file is auto-generated. If it differs from the previous one the CI will fail